src/lib/smartRefrigeratorLock.ts

```
@@ -0,0 +1,81 @@
+  return {
+    hash: `sha256_${hash}`,
+    expiresAt,
+  };
+}
+
+/**
+ * Caterer scans QR code on fridge to temporarily drop lock, deposit food, and trigger auto-lock (#4986).
+ */
+export function executeCatererQrDeposit(
+  fridgeId: string,
+  catererId: string = "u-caterer-101"
```

ESLint

[error] Insert `,`

(prettier/prettier)